### ISSUE TITLE
[Bug] Addons and Themes buttons missing in the settings menu

### ISSUE THREAD

**Issue by Absolute-Potato** [BetterDiscord/BetterDiscord#1653]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a plugin or theme

### Describe the Bug

I installed BD and these two settings tabs just aren't there. 
![image](https://github.com/BetterDiscord/BetterDiscord/assets/29511823/76af0bbc-52b7-4b5c-9dbc-a9dec4e97063)
Addons were definitely loading, or trying to load, because it notified me about addons being out of date from the remnants of an old installation. I deleted ./config/BetterDiscord/ and reinstalled from scratch, but its still not there.

Luckily I'm only installing it right now for custom CSS so its not the biggest problem for me, but being able to add plugins and change settings would be nice to have.

### Expected Behavior

The setting menus should all show up when installed

### Steps To Reproduce

0. Use Ubuntu 22.04
1. Download the appimage installer from the Installation step on the Github README
2. Install BD over your Discord stable install
3. Check user settings

### Operating System

Linux

### Discord Version

Stable

### BetterDiscord Version

v1.9.2

**Comment by rauenzi** [comment]
Your BD version is out of date. Please update and try again. Then check console and attach any errors.

**Comment by Absolute-Potato** [comment]
Oops, I think I just typo'd that. I double checked and it shows v1.9.3
![image](https://github.com/BetterDiscord/BetterDiscord/assets/29511823/7412d552-3391-4638-afc5-cdbad1d5b68b)

**Comment by Absolute-Potato** [comment]
I cannot stop fat fingering the wrong buttons lmao

**Comment by rauenzi** [comment]
As said, please check console for any errors

**Comment by Absolute-Potato** [comment]
[output.txt](https://github.com/BetterDiscord/BetterDiscord/files/12222439/output.txt)

**Comment by Absolute-Potato** [comment]
I was hoping that would embed, oh well.

~$ discord 
Discord 0.0.28
Starting app.
Starting updater.
2023-07-31T21:28:52.403Z [Modules] Modules initializing
2023-07-31T21:28:52.403Z [Modules] Distribution: remote
2023-07-31T21:28:52.403Z [Modules] Host updates: enabled
2023-07-31T21:28:52.403Z [Modules] Module updates: enabled
2023-07-31T21:28:52.403Z [Modules] Module install path: /home/potato/.config/discord/0.0.28/modules
2023-07-31T21:28:52.404Z [Modules] Module installed file path: /home/potato/.config/discord/0.0.28/modules/installed.json
2023-07-31T21:28:52.404Z [Modules] Module download path: /home/potato/.config/discord/0.0.28/modules/pending
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
2023-07-31T21:28:52.536Z [Modules] No updates to install
2023-07-31T21:28:52.536Z [Modules] Checking for host updates.
2023-07-31T21:28:52.614Z [Modules] Host is up to date.
2023-07-31T21:28:52.614Z [Modules] Checking for module updates at https://discord.com/api/modules/stable/versions.json
[85911:0731/172852.622632:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172852.623882:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172852.625125:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172852.626171:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172852.627207:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
2023-07-31T21:28:52.632Z [Modules] No module updates available.
Optional module ./ElectronTestRpc was not included.
[85911:0731/172852.706408:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172852.711684:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
2023-07-31T21:28:54.612Z [Modules] Checking for host updates.
2023-07-31T21:28:54.660Z [Modules] Host is up to date.
2023-07-31T21:28:54.661Z [Modules] Checking for module updates at https://discord.com/api/modules/stable/versions.json
2023-07-31T21:28:54.664Z [Modules] No module updates available.
[85911:0731/172855.162121:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172855.163101:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172855.164732:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172855.166027:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172855.168052:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172855.177807:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory
[85911:0731/172856.503889:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: BGRA_8888, share_between_threads: 0, gmb_type: shared_memory

**Comment by Absolute-Potato** [comment]
I honestly have no clue what happened but I went to take a shower and when I came back I realized it was still running via the terminal so I closed it and started it normally and it works now. The only thing I can think of is that I tried running it in wayland on the last terminal session? I don't think that should matter when I close it and launch it normally though... Anyways when it opened it popped up with a better discord what's new screen if that helps?
